### ISSUE TITLE
Таймеры

### ISSUE THREAD

**Issue by massed24** [tretyakovsa/Sonoff_WiFi_switch#101]
Всем привет, здорово, проект живёт и развивается, появилось много разных плюшек, возросла стабильность. Честь и хвала разработчикам! Обновился до последней бета версии через веб, кнопкой (SPIFFS&BUILD), появилась проблема с таймерами, таймер срабатывает и возвращается после нескольких секунд, то есть, реле включилось – секунда, две – выключилось. Обновлял таким образом в конце сентября, было все нормально.

**Comment by tretyakovsa** [comment]
@massed24 Сейчас проверю спасибо!

**Comment by massed24** [comment]
Обновил 10 минут назад ещё раз, через веб морду, теперь вообще не срабатывает. Надеюсь поправите, успехов, спасибо.

**Comment by tretyakovsa** [comment]
@massed24 Обновляйте еще раз. Должно все работать. Если что сообщайте.

**Comment by tretyakovsa** [comment]
@massed24 Да и после обновления делайте рефреш веб страницы на обновленном устройстве для очистки страниц в кеш браузера. ctrl+F5 в браузере.

**Comment by massed24** [comment]
Да, заработало, супер, правда не сразу, почему-то время NTP в настройках отключалось, по-новому сохранил, перезагрузил, сделал рефреш, забил таймер, и поехало как надо, будем тестировать дальше, спасибо за оперативность.

**Comment by massed24** [comment]
Привет, сегодня ещё не обновлялся, а вроде в субботу, так вот, с таймерами снова проблема, они выполняются один раз, хотя в настройках выставлено всегда.

**Comment by tretyakovsa** [comment]
Ошибку устранили пробуйте.

**Comment by massed24** [comment]
Судя по изменениям веб морды, вы сегодня несколько раз обновления выкладывали, удобней стало можно с любой страницы зайти куда угодно, перезагрузку упростили, добавили фишки. С таймерами не то что-то, первую ESP обновил, таймеры нормально заработали, через час обновил вторую ESP таймер отработал один раз, вместо всегда, потом снова обновил первую ESP и таймер снова отработал один раз. Мне это не критично, но мы ведь тестируем, думаю, сообщу, - не помешает.

**Comment by renat2985** [comment]
Спасибо за информацию.

**Comment by massed24** [comment]
Браво! исправили таймеры и пульсации, ну вы блин даёте.  :)
